src/installing-workflow/configuring-object-storage.md

```diff
@@ -111,4 +111,38 @@ The database looks for a `DATABASE_STORAGE` environment variable, which it then
 
 ## Credentials
 
-The database reads the credentials information from a `/var/run/secrets/deis/objectstore/creds/objectstorage-keyfile` file. This is generated automatically during helm generate based on the configuration options given in the https://github.com/deis/charts/blob/master/deis-dev/tpl/objectstorage.toml.
+Depending on the value of `DATABASE_STORAGE`, the database will either read the credentials from a generic objectstore secret or from a minio-user secret.in `/var/run/secrets/deis/objectstore/creds/` or from `/var/run/secrets/deis/database/creds/`. The following ways to configure the database are listed below.
+
+### Minio
+
+If the `DATABASE_STORAGE` backend is configured as anything else other than "s3", the database will receive its credentials from `/var/run/secrets/deis/database/creds/`. This is generated based on the configuration options given in the https://github.com/deis/charts/blob/master/workflow-dev/manifests/deis-minio-secret-user.yaml file. The access key and secret key must be `base64` encoded.
+
+Connection details to minio are configured via `DEIS_MINIO_SERVICE_HOST` and `DEIS_MINIO_SERVICE_PORT`, both of which are provided by the `deis-minio` service.
+
+### Amazon Simple Storage Service (S3)
+
+If the `DATABASE_STORAGE` backend is configured as "s3", the database will receive its credentials from `/var/run/secrets/deis/objectstore/creds/`. This is generated automatically (as part of the `helm generate` command) based on the configuration options given in the https://github.com/deis/charts/blob/master/workflow-dev/tpl/objectstorage.toml file.
+
+### Google Cloud Storage (S3 Compatibility Mode)
+
+If the `DATABASE_STORAGE` backend is configured as "gcs", the database will receive its credentials from `/var/run/secrets/deis/database/creds/`. This is generated based on the configuration options given in the https://github.com/deis/charts/blob/master/workflow-dev/manifests/deis-minio-secret-user.yaml file. The access key and secret key must be `base64` encoded.
+
+You'll also need to add two environment variables to the https://github.com/deis/charts/blob/master/workflow-dev/tpl/deis-database-rc.yaml file so the database can communicate with Google Cloud Storage instead of minio:
+
+```
+diff --git a/workflow-dev/tpl/deis-database-rc.yaml b/workflow-dev/tpl/deis-database-rc.yaml
+index e23fe45..c537b60 100644
+--- a/workflow-dev/tpl/deis-database-rc.yaml
++++ b/workflow-dev/tpl/deis-database-rc.yaml
+@@ -25,6 +25,10 @@ spec:
+           env:
+             - name: DATABASE_STORAGE
+               value: {{default "minio" .storage}}
++            - name: DEIS_MINIO_SERVICE_HOST
++              value: storage.googleapis.com
++            - name: DEIS_MINIO_SERVICE_PORT
++              value: "443"
+           readinessProbe:
+             exec:
+               command:
+```
```

Comment:
Can this be provided in a plain text format, instead of a diff? Also, since it shows the template syntax, I think this needs to mention that it needs to be installed with `helm generate && helm install`.